firebase-firestore/firebase-firestore.gradle

```diff
@@ -15,6 +15,9 @@
 plugins {
     id 'firebase-library'
     id 'com.google.protobuf'
+    id 'kotlin-android'
```

Comment:
We would also exclude kotlin stdlib( or rather move it to compileOnly), see how to disable it in https://kotlinlang.org/docs/gradle.html#dependency-on-the-standard-library

Reply:
1. the line: `id 'kotlin-android'`  is just a plugin, this is not affecting the runtime dependency size, right?
2. following your link, I have added a `kotlin.stdlib.default.dependency=false` to `gradle.properties`;
3. I added kotlin stdlib to be compile time only in `firebase-firstore.gradle`